v2/tools/generator/internal/astmodel/type_visitor_builder.go

```diff
@@ -66,20 +69,20 @@ func (b TypeVisitorBuilder) Build() TypeVisitor {
 // buildVisitTypeName returns a function to use in the TypeVisitor
 // If the field VisitTypeName is nil, we return an identity visitor. Otherwise we attempt to
 // convert the func found in the field, triggering a panic if no suitable func is found.
-func (b *TypeVisitorBuilder) buildVisitTypeName() func(*TypeVisitor, TypeName, interface{}) (Type, error) {
+func (b *TypeVisitorBuilder) buildVisitTypeName() (bool, func(*TypeVisitor, TypeName, interface{}) (Type, error)) {
```

Comment:
minor: maybe document the first bool return meaning in the functions // comments